release/cli/pkg/assets/config/bundle_release.go

```diff
@@ -58,6 +58,14 @@ var bundleReleaseAssetsConfigMap = []assettypes.AssetConfig{
 					ReleaseImageTagFormat:       "v<eksDReleaseChannel>-<eksDReleaseNumber>",
 				},
 			},
+			{
+				RepoName: "bottlerocket-bootstrap-vsphere-multi-network",
```

Comment:
We may consider removing vsphere from the image name everywhere as the bootstrap script itself doesn't do anything vsphere specific. Although we only support vsphere for bottlerocket as of now, it may be a good idea to keep it generic. I know it's a bigger change and requires changes in build-tooling repo as well. But it warrants a discussion as it might be more difficult to change this once we release to customers as it involves changes outward facing bundle crd and image uri changes.